### ISSUE TITLE
How to create new module

### ISSUE THREAD

**Issue by binSaed** [invoiceninja/admin-portal#364]
I followed "[flutter-redux-starter](https://github.com/hillelcoren/flutter-redux-starter#usage)" doc to create a new module

but I found the _starter.sh_ changed 

I tried to use admin-portal/starter.sh like this 
```
 chmod +x starter.sh

 # ./starter.sh make <application> <module_snake> <module_camel> <modules_snake> <modules_camel> <fields>

 ./starter.sh make invoiceninja_flutter room room rooms rooms title, url

 flutter packages pub run build_runner build --delete-conflicting-outputs
```

but I get a lot of errors

![Screen Shot 2021-11-04 at 10 47 37 PM](https://user-images.githubusercontent.com/33700292/140417573-014c37b1-e808-4618-b4c0-47fb3d722505.png)

### Plasse add some doc to let me continue my work (my boss will kick me out)😅

**Comment by hillelcoren** [comment]
Hi,

What are you plans with the app?

**Comment by AbdOoSaed** [comment]
Hi @hillelcoren,
We just need new module has the same functionality and same parameters of Product module

**Comment by hillelcoren** [comment]
I just want to confirm this is for your own use and you aren't planning to resell the app.

**Comment by AbdOoSaed** [comment]
@hillelcoren 
Yes it will be my own use and we aren't planning to sell it, my company will use it internally to manage some products

**Comment by hillelcoren** [comment]
Thanks, sounds good!

I think you may be able to correct some of the errors by uncommenting this line and trying again from a clean start.

https://github.com/invoiceninja/admin-portal/blob/master/starter.sh#L159

The module generator can break over time as we make other changes in the app, we'll look into correcting it.
